Tests/SonarScanner.MSBuild.PreProcessor.Test/CacheProcessorTests.cs

```diff
@@ -92,6 +94,9 @@ private static string CreateFile(string root, string fileName, string content, E
             return path;
         }
 
+        private static string Serialize(byte[] value) =>
```

Comment:
I would rename this `ToString` (maybe an extension method on `byte[]`?). I think the code will be easier to read/understand.

Reply:
`byte[]` has `ToString` so we would not get invoked :/

It's used only in one method, will making it a local function help? (I can do it in next PR, I need to merge this for rebasing)

Reply:
Sure, It can be a local function.